src/Transfer/Resources/Database/Attributes/DateTimeAttribute.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class DateTimeAttribute extends Attribute
```

Comment:
```suggestion
class DateTime extends Attribute
```
No need of the Attribute suffix. Make sure to change it in the file name.